docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
FWIW the concept reminds me of [workers](https://developer.mozilla.org/en-US/docs/Web/API/Worker).